### PR TITLE
PR: French translation for 3.2.7

### LINKED ISSUE #6484: Update translations for 3.2.7
Version **3.2.7** is finally ready, so now it has come the time to update our translations for this release.

This is a call to all our translators to update their respective translations. Please create separate pull requests for each language:

- French
  - @goanpeca 
  - @Prikers 
- Japanese
  - @sohnishi
- Portuguese (Brazilian)
  - @manipuladordedados
- Russian
  - @burrbull
  - @Qlogin
- Spanish
  - @ccordoba12

---

**Important notes**:

1. We can only wait for you until Wednesday February 28 at 00:00 am UTC for your pull requests. There are 17 strings to be updated or translated.
2. Don't forget to do your PRs against the **3.x** branch.

### PR DESCRIPTION
**Pull request by Prikers**
Fixes #6484 (for the French part)

### CODE REVIEW

File: spyder/locale/fr/LC_MESSAGES/spyder.po [hunk 1]
```diff
@@ -4040,21 +4042,20 @@ msgid "Project"
...
+"projet en cours (si ouvert)"
 
 #: spyder/widgets/findinfiles.py:236
 msgid "File"
 msgstr "Fichier"
 
 #: spyder/widgets/findinfiles.py:237
 msgid "Search in current opened file"
-msgstr "Recherche dans le fichier ouvert actuel"
+msgstr "Rechercher dans le fichier en cours"
```
Comment (goanpeca) [inline spyder/locale/fr/LC_MESSAGES/spyder.po]: Hmm intersting :-p

File: spyder/locale/fr/LC_MESSAGES/spyder.po [hunk 2]
```diff
@@ -4351,7 +4348,7 @@ msgstr "Le noyau est mort, redémarrage en cours"
 
 #: spyder/widgets/ipythonconsole/shell.py:475
 msgid "Kernel restarting"
-msgstr "Redémarrage du kernel"
+msgstr "Redémarrage du noyau"
```
Comment (goanpeca) [inline spyder/locale/fr/LC_MESSAGES/spyder.po]: Is this something used in French?

Comment (Prikers) [reply spyder/locale/fr/LC_MESSAGES/spyder.po]: yes :-D

**Comment by ccordoba12** [comment]
Thanks @Prikers!

**Pull request merged**